internal/project/auto/markdown.go

```diff
@@ -22,23 +23,25 @@ func (builder *builder) DetectMarkdown() (bool, error) {
 		}
 	}
 
-	{
-		list, err := listFilesWithSuffix(builder.rootPath, ".md")
-		if err != nil {
-			return false, err
-		}
+	list, err := listFilesWithSuffix(builder.rootPath, ".md")
```

Comment:
this was just cleanup, the previous code was wrapped in `{}`

Reply:
it was wrapped for readability, probably unsuccessful attempt to make it more readable :stuck_out_tongue_winking_eye: